src/action.js

```diff
@@ -255,19 +280,61 @@ async function setStatus(pullRequest, state, description) {
   });
 }
 
+async function generateReleaseNotes(pullRequest) {
+  const tag = getTagName(pullRequest);
+
+  var previous_tag_name;
+  try {
+    const latest_release = await client.rest.repos.getLatestRelease({
```

Comment:
Just a tiny nitpick, but we should align the naming here with the convention we have in this project, i.e. camel case for variable/function names.

```suggestion
    const latestRelease = await client.rest.repos.getLatestRelease({
```